libparsec/crates/client/src/certificates_ops/add.rs

```diff
@@ -102,7 +102,7 @@ pub enum AddCertificateError {
     Internal(#[from] anyhow::Error),
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
please staaaap :'(

Reply:
Done

Reply:
That's why I don't like to open multiple PR, when someone review something, I have to change everywhere